package.json

```diff
@@ -18,6 +18,7 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.54",
     "@material-ui/pickers": "^3.2.10",
+    "@material-ui/styles": "^4.11.5",
```

Comment:
Why did you need to add this package?

Reply:
There was an error raised by material-ui and I found out that installing styles would solve the issue. I'm going to uninstall it though because we have since made quite a lot of changes